aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -193,6 +257,8 @@ module aptos_framework::genesis {
         chain_status::set_genesis_end(aptos_framework);
     }
 
+    native fun create_signer(addr: address): signer;
```

Comment:
Nice!

Reply:
hahaha

Reply:
do you need this? or let account friend genesis is enough?

Reply:
wanted to limit friending all over the place...